### PR TITLE
Member check

### PR DESCRIPTION
**Pull request by openorclose**
Check array/ object property getting/setting.

### CODE REVIEW

File: src/transpiler.ts [hunk 1]
```diff
@@ -403,6 +405,44 @@ function transformUnaryAndBinaryOperationsToFunctionCalls(program: es.Program) {
...
 }
 
+function transformPropertyAssignment(program: es.Program) {
+  simple(program, {
+    AssignmentExpression(node) {
+      const assignmentExpression = node as es.AssignmentExpression
+      if (assignmentExpression.left.type === 'MemberExpression') {
+        const { object, property, computed, loc } = assignmentExpression.left
+        const { line, column } = loc!.start
+        node = node as es.CallExpression
```
Comment (thomastanck) [inline src/transpiler.ts]: Is it possible to use create.mutateToCallExpression here?

File: src/transpiler.ts [hunk 2]
```diff
@@ -403,6 +405,44 @@ function transformUnaryAndBinaryOperationsToFunctionCalls(program: es.Program) {
...
+  })
+}
+
+function transformPropertyAccess(program: es.Program) {
+  simple(program, {
+    MemberExpression(node) {
+      const memberExpression = node as es.MemberExpression
+      const { object, property, computed, loc } = memberExpression
+      const { line, column } = loc!.start
+      node = node as es.CallExpression
```
Comment (thomastanck) [inline src/transpiler.ts]: Same.

Comment (openorclose) [reply src/transpiler.ts]: Same.

File: src/transpiler.ts (hunk 1 above)
Comment (openorclose) [reply src/transpiler.ts]: Yes. Seems like I made it because it was horrible to change something to callExpression, but forgot all about it.

**Pull request merged**